packages/manager/apps/freefax/package.json

```diff
@@ -35,7 +35,7 @@
     "oclazyload": "^1.1.0",
     "ovh-angular-ui-confirm-modal": "^1.0.2",
     "ovh-api-services": "^6.23.0",
-    "ovh-ui-angular": "^3.2.1",
+    "ovh-ui-angular": "^3.2.0",
```

Comment:
Should be `3.2.2` right ?

Reply:
i prefer ^3.2.0 since it includes 3.2.2 and further future patches 3.2.x, what do you think?

Reply:
Depends on your needs!

Maybe you need a fix in ovh-ui-angular which means that you need at least the version 3.2.2. `^3.2.2` is a good compromise ;D